modules/edu.gemini.seqexec.engine/src/main/scala/edu/gemini/seqexec/engine/package.scala

```diff
@@ -275,12 +275,12 @@ package object engine {
     /**
       * Log info lifted into Handle.
       */
-    def info(msg: String): HandleP[Unit] = pure((logger.info(msg), None))
+    def info(msg: String): HandleP[Unit] = pure({(logger.info(msg), None); ()})
```

Comment:
Might also work to use `scalaz.Functor.void` if necessary as in `pure((logger.info(msg), None)).void`.